src/magic.cpp

```diff
@@ -1066,6 +1066,10 @@ void spellcasting_callback::draw_spell_info( const spell &sp, const uilist *menu
     } else if( fx == "spawn_item" ) {
         damage_string = _( string_format( "Spawn %d %s", sp.damage(), item::nname( sp.effect_data(),
                                           sp.damage() ) ) );
+    } else if( fx == "summon" ) {
+        damage_string = _( string_format( "Summon %d %s", sp.damage(),
```

Comment:
Translation an already formatted string won't work. You need to translate the parts separately. Same below.